docs/collecting-data/collecting-from-own-applications/mobile-trackers/remote-configuration/index.md

```diff
@@ -45,7 +45,11 @@ let successCallback: ([String]?) -> Void = { namespaces, state in
 }
 
 // Set up tracker with remote configuration
-Snowplow.setup(remoteConfiguration: remoteConfig, defaultConfiguration: defaultConfig, onSuccess: successCallback)
+Snowplow.setup(
+  remoteConfiguration: remoteConfig,
+  defaultConfiguration: defaultConfig,
+  defaultConfigurationVersion: 1,
+  onSuccess: successCallback)
```

Comment:
nit: dunno what's correct Swift style, should this closing bracket be on the line below?